tools/azure-sdk-tools/tests/test_pyproject_interactions.py

```diff
@@ -0,0 +1,83 @@
+import os, tempfile, shutil
+
+
+import pytest
+
+from ci_tools.parsing import update_build_config, get_build_config, get_config_setting
+
+integration_folder = os.path.join(os.path.dirname(__file__), "integration")
+pyproject_folder = os.path.join(integration_folder, "scenarios", "sample_pyprojects")
+pyproject_file = os.path.join(integration_folder, "scenarios", "sample_pyprojects", "pyproject.toml")
+
+
+@pytest.mark.parametrize(
+    "target",
+    [
+        # check is true
+        # pyproject_folder,
+        # pyproject_folder,
```

Comment:
Looks like this is supposed to be uncommented, with one being a reference to the `pyproject` file? This would obviate the following `test_pyproject_file_parse` test though, so I could be misunderstanding the intent
```suggestion
        pyproject_file,
        pyproject_folder,
```

Reply:
yep exactly. I had it narrowed down to fix the last option 😂